### ISSUE TITLE
Env variables problem with Travis on PULL request from FORK

### ISSUE THREAD

**Issue by LEQADA** [sphereio/sphere-node-sdk#191]
The problem is that if user has no PUSH access to the repository then he need to FORK repository and then create a PULL request, and in this case Travis will not decrypt encrypted environment variables from .yaml file for security reasons.
This feature [described in documentation of Travis](https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml)

> Encrypted environment variables are not available to pull requests from forks due to the security risk of exposing such information to unknown code.

I researched bit and found this [JWT solution](https://docs.travis-ci.com/user/jwt) from Travis.
The main idea is to create temporary tokens instead of encrypted env variables.
Basically how it works:
1. You create token from your env variables using travis tool
2. Write that token to .yaml file
3. Make a PULL request from your FORK
4. Travis will get your token and replace it with your env variables
5. Travis will execute tests with your env variables
6. After 90 mins your token will die

More about it you can find in the [documentation of JWT addon](https://docs.travis-ci.com/user/jwt)

**Comment by emmenko** [comment]
Discussion has been moved to https://github.com/commercetools/nodejs/issues/30

Closing this.
